### PR TITLE
allow modular_koji_tags to be boolean in schema

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-1241

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
Since this behavior is not supported in ODCS (passing a boolean to modular_koij_tags), This should be well documented here and in the osbs-docs.

The description should be updated to explicitly say what setting he value to True means and how it differs from setting it to a list, having a null value, or setting it to False.

**Comment by rcerven** [comment]
> Since this behavior is not supported in ODCS (passing a boolean to modular_koij_tags), This should be well documented here and in the osbs-docs.
> 
> The description should be updated to explicitly say what setting he value to True means and how it differs from setting it to a list, having a null value, or setting it to False.

it is documented in https://github.com/containerbuildsystem/osbs-docs/pull/158/files

File: osbs/schemas/container.json [hunk 1]
```diff
@@ -241,8 +241,8 @@
           }
         },
         "modular_koji_tags": {
-          "description": "tags of modular Koji content to include in the build",
-          "type": ["array", "null"],
+          "description": "tags of modular Koji content to include in the build, if True is used, tag from target will be used",
```
Comment (athos-ribeiro) [inline osbs/schemas/container.json:244]: Thnks for changing this!

I'd be even more explicit here to say `tag from koji target will be used`!

Comment (rcerven) [reply osbs/schemas/container.json]: updated

**Comment by rcerven** [comment]
rn updated: allow modular_koji_tags to be boolean in schema

**Pull request merged**